sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -530,6 +530,9 @@ async def _read(self, n, buffer=None, **kwargs):  # pylint: disable=unused-argum
             return view
         except asyncio.TimeoutError as te:
             raise ConnectionError('recv timed out (%s)' % te)
+        except OSError:
+            await self.session.close()
+            raise ConnectionError('Client Session Closed')
```

Comment:
Could you please include underlying error in the raise one? And I think the phrase "client session" might be confusing.
e.g.:
```python
except OSError as e:
    await self.session.close()
    raise ConnectionError('Websocket connection closed: %r' % e) from e
```